redbeams/src/main/java/com/sequenceiq/redbeams/flow/redbeams/start/handler/CertRotateInRedBeamsHandler.java

```diff
@@ -0,0 +1,40 @@
+package com.sequenceiq.redbeams.flow.redbeams.start.handler;
+
+import static org.slf4j.LoggerFactory.getLogger;
+
+import org.slf4j.Logger;
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.cloudbreak.common.event.Selectable;
+import com.sequenceiq.cloudbreak.eventbus.Event;
+import com.sequenceiq.flow.event.EventSelectorUtil;
+import com.sequenceiq.flow.reactor.api.handler.ExceptionCatcherEventHandler;
+import com.sequenceiq.flow.reactor.api.handler.HandlerEvent;
+import com.sequenceiq.redbeams.flow.redbeams.start.event.CertRotateInRedBeamsRequest;
+import com.sequenceiq.redbeams.flow.redbeams.start.event.CertRotateInRedBeamsSuccess;
+import com.sequenceiq.redbeams.flow.redbeams.start.event.StartDatabaseServerFailed;
+
+@Component
+public class CertRotateInRedBeamsHandler extends ExceptionCatcherEventHandler<CertRotateInRedBeamsRequest> {
+
+    private static final Logger LOGGER = getLogger(CertRotateInRedBeamsHandler.class);
+
+    @Override
+    public String selector() {
+        return EventSelectorUtil.selector(CertRotateInRedBeamsRequest.class);
+    }
+
+    @Override
+    protected Selectable defaultFailureEvent(Long resourceId, Exception e, Event<CertRotateInRedBeamsRequest> event) {
+        LOGGER.error("Cannot rotate the root CERT, error: ", e);
+        return new StartDatabaseServerFailed(resourceId, e);
```

Comment:
I think we need a dedicated failure type; `StartDatabaseServerFailed` is associated with `StartDatabaseServerHandler`.

Reply:
I have changed the failure vent to `START_DATABASE_SERVER_FAILED_EVENT` so it will handle the failure correctly with this class